.openshiftio/application.yaml

```diff
@@ -10,6 +10,14 @@ metadata:
     description: >-
       The REST API Level 0 Mission provides a basic example of mapping business operations to a remote procedure call endpoint over HTTP using a REST framework.
 parameters:
+- name: SUFFIX_NAME
+  description: The suffix name for the template objects
+  displayName: Suffix name
+  value: ''
+- name: RELEASE_VERSION
+  description: The release version number of application
+  displayName: Release version
+  value: 2.0.0
```

Comment:
Not sure how OSIO creates these numbers, but why start from `2.0.0`?

Reply:
I have made it 1.0.0